sdks/python/apache_beam/typehints/trivial_inference.py

```diff
@@ -398,19 +444,40 @@ def infer_return_type_func(f, input_types, debug=False, depth=0):
       else:
         return_type = Any
       state.stack[-pop_count:] = [return_type]
-    elif (opname == 'BINARY_SUBSCR'
```

Comment:
Why was this one removed?

Reply:
I moved the code into opcodes.py. Also, I believe the check of `state.stack[1]` and `state.stack[0]` was buggy (should have subscripted `[-1]` and `[-2]`).